src/python/grpcio_tests/tests/fork/_fork_interop_test.py

```diff
@@ -34,23 +47,33 @@
     from grpc._cython import cygrpc
```

Comment:
Won't let me comment on previous lines but looks like we don't need `import sys` here?

Reply:
Ah, yes. Another missed debug line. Nice catch!